server/controllers/refer/referralsController.test.ts

```diff
@@ -236,44 +236,6 @@ describe('ReferralsController', () => {
     })
```

Comment:
I think the `update` action needs updating in this (second) commit too (to remove the `oasysConfirmation` handling, and remove the related test)

Reply:
That's going to come in a future PR:
> N.B: I've left the tests and functionality for the ReferralController's update method in place here, as I didn't want to remove them until we'd moved the reason-setting behaviour into its own controller, but they will be removed in future.

Reply:
Hmm okay. I was thinking something like [this from the reason PR](https://github.com/ministryofjustice/hmpps-accredited-programmes-ui/pull/199/commits/05f100011e38703149cff0c67a8a19717a522fea) since the `oasysConfirmed` update functionality is no longer used with these changes, but it could wait for a separate cleanup PR

Reply:
I suppose depending on who's PR is merged first we can remove this in the later branch